pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -231,13 +231,16 @@ func newLogicalReplicationWriterProcessor(
 		checkpoint:  lrw.checkpoint,
 		bytesGauge:  lrw.metrics.RetryQueueBytes,
 		eventsGauge: lrw.metrics.RetryQueueEvents,
+		debug:       lrw.debug,
 	}
 
 	if err := lrw.Init(ctx, lrw, post, logicalReplicationWriterResultType, flowCtx, processorID, nil, /* memMonitor */
 		execinfra.ProcStateOpts{
 			InputsToDrain: []execinfra.RowSource{},
 			TrailingMetaCallback: func() []execinfrapb.ProducerMetadata {
+				log.Infof(lrw.Ctx(), "logical replication writer processor closing")
```

Comment:
I think we'd be fine with just one of these two lines?

Reply:
Removed the second one.